test/infrastructure/container/docker.go

```diff
@@ -331,8 +332,13 @@ func (d *dockerRuntime) ContainerDebugInfo(ctx context.Context, containerName st
 		return errors.Wrapf(err, "failed to inspect container %q", containerName)
 	}
 
+	rawJSON, err := json.Marshal(containerInfo)
+	if err != nil {
+		return errors.Wrapf(err, "Marshal container info to json")
```

Comment:
```suggestion
		return errors.Wrapf(err, "failed to marshal container info to json")
```
nit: just for lower case + to align to the general error style around here